src/modules/Verifications/components/VerificationList.tsx/index.tsx

```diff
@@ -80,40 +80,36 @@ const VerificationList: React.FC<VerificationListProps> = ({ list, totalItems })
   return (
     <>
       <div className={css['table']}>
-        <div className="block overflow-auto">
-          <table className="w-full rounded-lg">
-            <thead className={css['header']}>
-              {table.getHeaderGroups().map(headerGroup => {
-                return (
-                  <tr key={headerGroup.id}>
-                    {headerGroup.headers.map(header => {
-                      return (
-                        <th id={header.id} key={header.id} className={css['header__item']}>
-                          {header.isPlaceholder
-                            ? null
-                            : flexRender(header.column.columnDef.header, header.getContext())}
-                        </th>
-                      );
-                    })}
-                  </tr>
-                );
-              })}
-            </thead>
-            <tbody>
-              {table.getRowModel().rows.map(row => {
-                return (
-                  <tr key={row.id} className="cursor-pointer">
-                    {row.getVisibleCells().map(cell => (
-                      <td className={css['col']} key={cell.id}>
-                        {flexRender(cell.column.columnDef.cell, cell.getContext())}
-                      </td>
-                    ))}
-                  </tr>
-                );
-              })}
-            </tbody>
-          </table>
-        </div>
+        <table className="w-full rounded-lg">
```

Comment:
removing overflow caused prevent scrolling in different breakpoints

Reply:
Based on design, table does not have scroll on overflow, instead the page scrolls in case of long rows